### DISCUSSION TITLE
Outgoing Emails stuck pending in queue

### DISCUSSION THREAD

**Discussion by arjwalker** [postalserver/postal#2385] [Help with using Postal] [answered]
Hi All,

I'm new to Postal (and Linux) and trying to get outgoing SMTP up and running to replace a paid-for service we are currently using. 

All was looking great until testing and all emails outgoing are sitting in "pending"

I've added a domain with all green (I've not set MX as it won't be used for incoming) 
Activity shows the message received by Postal but nothing else

Ive tested from external SMTP and using the built-in Send Message but its the same outcome. 
Ive enabled advanced SMTP Logs which doesn't seem to show much apart from this:

![image](https://user-images.githubusercontent.com/91224074/226420034-4dcb1a55-d93e-4492-8cca-98e2cd34dcce.png)

I don't recognise that IP either! 

Ive just noticed that checking postal status the requeuer and worker are in Exit 1 state which they were not before? 
![image](https://user-images.githubusercontent.com/91224074/226421197-f8c7ca6a-519a-4b11-939f-3a6fd9e702bd.png)

Does anybody have an idea where to start to investigate, is there a way to see a live log rather than loading this entire log every time too? 

Any help would be greatly appreciated! 
Thanks!

**Comment by arjwalker** [comment, marked as answer]
Just to update and possibly help others in the future, it would seem that running `postal logs worker` showed that the RabbitMQ was not authenticating I removed the container and re-added a password with no symbols and when I reloaded it all connected and mail started to flow!

**Reply by willpower232** [reply]
glad you figured it out, you can check on how postal is doing with `postal status` but yes it is usually rabbitmq having a bad time that stops the workers working.
